WordPressAuthenticator/Analytics/SignInTracker.swift

```diff
@@ -0,0 +1,341 @@
+        
+        /// This flow represents the signup (when the user inputs an email that’s not registered with a .com account)
+        ///
+        case signup
+    }
+    
+    enum Step: String {
+        /// Gets shown on the Prologue screen
+        ///
+        case prologue
+        
+        /// Triggered when a flow is started
+        ///
+        case start
+        
+        /// Triggered when a user requests a magic link and sees the screen with the “Open mail” button
+        ///
+        case magicLinkRequested = "magic_link_requested"
+        
+        /// This represents the user opening their mail. It’s not strictly speaking an in-app screen but for the user it is part of the flow.
+        case emailOpened = "email_opened"
+        
+        /// The screen with a username and password visible
+        ///
+        case userPasswordScreenShown = "username_password"
+        
+        /// Triggered on the epilogue screen
+        ///
+        case success
+        
+        /// Triggered on the help screen
+        ///
+        case help
+        
+        /// When we ask user to input the code from the 2 factor authentication
+        case twoFactorAuthentication = "2fa"
+    }
+    
+    enum ClickTarget: String {
+        /// Tracked when submitting the email form, the email & password form, site address form,
+        /// username & password form and signup email form
+        ///
+        case submit
+        
+        /// Tracked when the user clicks on continue in the login/signup epilogue
+        ///
+        case `continue`
+        
+        /// Tracked when the post signup interstitial screen is dismissed, when the
+        /// login signup help dialog is dismissed and when the email hint dialog is dismissed
+        ///
+        case dismiss
+        
+        /// Tracked when the user clicks “Continue with WordPress.com” on the Prologue screen
+        ///
+        case continueWithWordPressCom = "continue_with_wordpress_com"
+        
+        /// Tracked when the user clicks “Login with site address” on the Prologue screen
+        ///
+        case loginWithSiteAddress = "login_with_site_address"
+        
+        /// Tracked when the user clicks “Login with Google” on the WordPress.com flow screen
+        ///
+        case loginWithGoogle = "login_with_google"
+        
+        /// When the user clicks on “Forgotten password” on one of the screens that show the password field
+        ///
+        case forgottenPassword = "forgotten_password"
+        
+        /// When the user clicks on terms of service anywhere
+        ///
+        case termsOfService = "terms_of_service_clicked"
+        
+        /// When the user tries to sign up with email from the confirmation screen
+        ///
+        case signUpWithEmail = "signup_with_email"
+        
+        /// When the user tries to sign up with Google from the confirmation screen
+        ///
+        case signUpWithGoogle = "signup_with_google"
```

Comment:
I think `signUpWithApple` needs added above this?

Reply:
Good catch!  I think this was missing mostly because the list I'm using is shared with Android.